cw/capture.py

```diff
@@ -135,7 +148,11 @@ def capture_init(ctx, num_traces, plot_traces):
 
     # Key and plaintext generator
     ctx.obj.ktp = cw.ktp.Basic()
-    ctx.obj.ktp.key_len = cfg["capture"]["key_len_bytes"]
+    # This is a workaroung for https://github.com/lowRISC/ot-sca/issues/116
+    if "use_fixed_key_iter" in cfg["capture"]:  # for backwarts compatibility
+        ctx.obj.ktp.fixed_key = cfg["capture"]["use_fixed_key_iter"]
+        # if fixed_key = true, then key never updates and setting...
+    ctx.obj.ktp.key_len = cfg["capture"]["key_len_bytes"]  # ...key_len has no effect
```

Comment:
nit: This is a bit strange way of writing comments. I would avoid having code and comment in the same line.